server/main.go

```diff
@@ -58,7 +58,9 @@ func main() {
 
 	r.HandleFunc("/project", mkHandler(ctx.getProjects)).Methods("GET")
 	r.HandleFunc("/project/{id}", mkHandler(ctx.getProject)).Methods("GET")
-	r.HandleFunc("/project/{id}/job", mkHandler(ctx.startBuild)).Methods("POST")
+	r.HandleFunc("/project/{id}/job", mkHandler(ctx.startStandardJob)).Methods("POST")
```

Comment:
We could. We need to differentiate https://confluence.atlassian.com/display/BITBUCKET/POST+hook+management from  https://developer.github.com/v3/activity/events/types/#pushevent and our classic json data

But I think to introduce pluggable hook endpoints in the future, it would be better that each hoster has its own endpoint